pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md

```diff
@@ -70,298 +71,510 @@ us-west1
+    ```hcl
+    variable "dcos_install_mode" {
+      description = "specifies which type of command to execute. Options: install or upgrade"
+      default = "install"
+    }
 
-Set this parameter to `upgrade` during upgrade process.
+    data "http" "whatismyip" {
+      url = "http://whatismyip.akamai.com/"
+    }
 
-<p class="message--important"><strong>IMPORTANT: </strong>Do not change any number of masters, agents, or public agents while performing an upgrade.</p>
+    module "dcos" {
+      source = "dcos-terraform/dcos/gcp"
 
-```hcl
-variable "dcos_install_mode" {
-  description = "specifies which type of command to execute. Options: install or upgrade"
-  default = "install"
-}
+      cluster_name        = "my-open-dcos"
+      ssh_public_key_file = "~/.ssh/id_rsa.pub"
+      admin_ips           = ["${data.http.whatismyip.body}/32"]
 
-data "http" "whatismyip" {
-  url = "http://whatismyip.akamai.com/"
-}
+      num_masters        = "1"
+      num_private_agents = "3"
+      num_public_agents  = "1"
 
-module "dcos" {
-  source = "dcos-terraform/dcos/gcp"
+      dcos_version = "1.11.5"
 
-  cluster_name        = "my-open-dcos"
-  ssh_public_key_file = "~/.ssh/id_rsa.pub"
-  admin_ips           = ["${data.http.whatismyip.body}/32"]
+      # dcos_variant              = "ee"
+      # dcos_license_key_contents = "${file("./license.txt")}"
+      dcos_variant = "open"
 
-  num_masters        = "1"
-  num_private_agents = "3"
-  num_public_agents  = "1"
+      dcos_install_mode = "${var.dcos_install_mode}"
+    }
 
-  dcos_version = "1.11.4"
+    output "masters-ips" {
+      value       = "${module.dcos.masters-ips}"
+    }
 
-  # dcos_variant              = "ee"
-  # dcos_license_key_contents = "${file("./license.txt")}"
-  dcos_variant = "open"
+    output "cluster-address" {
+      value       = "${module.dcos.masters-loadbalancer}"
+    }
 
-  dcos_install_mode = "${var.dcos_install_mode}"
-}
+    output "public-agents-loadbalancer" {
+      value = "${module.dcos.public-agents-loadbalancer}"
+    }
+    ```
 
-output "masters-ips" {
-  value       = "${module.dcos.masters-ips}"
-}
+3) Re-run the new execution plan.  
 
-output "cluster-address" {
-  value       = "${module.dcos.masters-loadbalancer}"
-}
+    ```bash
+    terraform plan -out=plan.out -var dcos_install_mode=upgrade
+    ```
 
-output "public-agents-loadbalancer" {
-  value = "${module.dcos.public-agents-loadbalancer}"
-}
-```
+    You should see output similar to the following:
+  <p align=center><img src="./images/upgrade/terraform-plan.png" /></p>
 
-2) Re-run our execution plan.  
+4) Run the following command to apply the plan.
 
-```bash
-terraform plan -out=plan.out -var dcos_install_mode=upgrade
-```
-
-You should see an output like below.
+    ```bash
+    terraform apply plan.out
+    ```
 
-<p align=center>
-<img src="./images/upgrade/terraform-plan.png" />
-</p>
+5) Verify the cluster was upgraded using the DC/OS UI.
```

Comment:
Once the `terraform apply` completes, verify that your DC/OS cluster was upgraded via the UI.